package.json

```diff
@@ -45,12 +45,15 @@
     "sinon": "^11.1.1",
     "tslib": "^2.1.0"
   },
+  "peerDependencies": {
+    "@aws-sdk/types": "^3.0.0"
```

Comment:
We should allow any aws-sdk/type above 3.0.0 that the customer wants to bring. This is should be resolved at a higher tree node when people `npm install`